nativelib/src/main/scala/scala/scalanative/meta/LinktimeInfo.scala

```diff
@@ -8,9 +8,15 @@ import scala.scalanative.unsafe._
 object LinktimeInfo {
   @resolvedAtLinktime("scala.scalanative.meta.linktimeinfo.isWindows")
   def isWindows: Boolean = resolved
+  
+  @resolvedAtLinktime("scala.scalanative.meta.linktimeinfo.isMac")
+  def isMac: Boolean = resolved
```

Comment:
Linktime resolved values/conditions were designed to dead code eliminate untaken branches at link time, and the main goal was to get rid of usages of native functions that are not available in Windows/Unix. MacOS does not define any os-specific functions that would not be available in other UNIX based operating systems. 
We want to limit the amount of link-time resolved values to a minimum, and use runtime resolved values whenever possible. 
Usage of this method can be replaced with `scala.scalanative.runtime.Platform.isMac`

Reply:
Gotcha, thanks